specs/_template/02_user_interface_chrome.rst

```diff
@@ -0,0 +1,32 @@
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
+
+===========================
+User interface (Chrome)
+===========================
+
+.. contents:: Table of Contents
+
+1. SCREEN_NAME
```

Comment:
Should screen names be in all caps?

Reply:
no, this is a template variable. I see the intention is not clear here.

Reply:
ok